clients/src/main/java/org/apache/kafka/common/protocol/types/Type.java

```diff
@@ -102,6 +108,13 @@ public Boolean validate(Object item) {
             else
                 throw new SchemaException(item + " is not a Boolean.");
         }
+
+        @Override
+        public String documentation() {
+            return "Represents a boolean value in a byte. " +
+                    "Values 0 and 1 is used to represent false and true respectively. " +
+                    "When reading a Boolean value, any non-zero value is considered true.";
```

Comment:
nit: not sure why Boolean is capitalized here, but not above.

Reply:
I think the first instance referred to the concept of booleans while the second referred to this particular type. But the distinction might only live in my head so I use lowercase consistently instead.